src/routes/productRouter.ts

```diff
@@ -2,12 +2,13 @@ import {Router} from "express"
 import productController from "../modules/product/controller/productController";
 import { userAuthorization } from "../middlewares/authorization";
 import { validation, isProductExist, isShopExist, transformFilesToBody } from "../middlewares/validation";
-import { shopSchema, productSchema} from "../modules/product/validation/productValidation";
+import { shopSchema, productSchema, availabilitySchema} from "../modules/product/validation/productValidation";
 import upload from "../helpers/multer";
 
 const router: Router = Router()
 
 router.post("/create-product", userAuthorization(["seller"]), upload.array("images"), transformFilesToBody,validation(productSchema), isProductExist, productController.createProduct);
 router.post("/create-shop", userAuthorization(["seller"]), validation(shopSchema), isShopExist, productController.createShop)
+router.put("/update-product-availability/:id", userAuthorization(["seller"]), validation(availabilitySchema), productController.updateProductAvailability);

```

Comment:
Rename from **`/update-product-availability/:id`** to **`/update-product-status/:id`**